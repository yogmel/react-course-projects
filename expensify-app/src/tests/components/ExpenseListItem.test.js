import React from 'react'
import { shallow } from 'enzyme'
import ExpenseListitem from '../../components/ExpenseListItem'
import expenses from '../fixtures/expenses'

test('should render a single expense item', () => {
  const wrapper = shallow(<ExpenseListitem {...expenses[0]}/>)
  expect(wrapper).toMatchSnapshot()
})