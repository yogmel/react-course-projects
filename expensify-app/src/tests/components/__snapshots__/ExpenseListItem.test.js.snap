// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a single expense item 1`] = `
<div>
  <h1>
    <Link
      replace={false}
      to="/edit/1"
    >
      Gum
    </Link>
  </h1>
  <p>
     
    195
     - 
    0
     
  </p>
</div>
`;
